studio/pages/project/[ref]/settings/database.tsx

```diff
@@ -303,14 +303,18 @@ const GeneralSettings: FC<any> = ({ projectRef }) => {
   const DB_FIELDS = ['db_host', 'db_name', 'db_port', 'db_user', 'inserted_at']
   const connectionInfo = pluckObjectFields(formModel, DB_FIELDS)
 
-  const defaultConnString =
+  const defaultUriConnString =
```

Comment:
let's remove the `default`, does not make sense in this context anymore.

Reply:
okay